src/zm_config.h.in

```diff
@@ -65,6 +65,8 @@ struct StaticConfig
     std::string DB_USER;
     std::string DB_PASS;
     std::string PATH_WEB;
+	std::string SERVER_NAME;
+	unsigned int SERVER_ID;
```

Comment:
Well, for the average person yeah... but what about when we take over the world and literally have millions of zm servers all over the internet to manage?